AEPConcierge/Sources/ConciergeChatViewModel.swift

```diff
@@ -218,13 +218,27 @@ final class ConciergeChatViewModel: ObservableObject {
                         
                         let state = payload.state
                         
+                        if let response = payload.response {
+                            let text = response.message
+                            let srcCount = response.sources?.count ?? 0
+                            let suggCount = response.promptSuggestions?.count ?? 0
```

Comment:
nit: no need to make variables here since we aren't using them anywhere other than the log